src/styles/main.scss

```diff
@@ -10,5 +14,3 @@ body {
   box-sizing: border-box;
 }
```

Comment:
I would transfer the main page styles to a separate file - normalize.scss or reset.scss